src/Components/ComponentPreviewTransformModel.cs

```diff
@@ -0,0 +1,74 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ComponentPreviewTransformModel.cs" company="GlobalLink Vasont">
+// Copyright (c) GlobalLink Vasont. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Vasont.Inspire.Models.Components
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System;
+
+    /// <summary>
+    /// Contains an enumerated list of process types.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ComponentProcessType
+    {
+        /// <summary>
+        /// Contains a value that indicates the processtype is view.
+        /// </summary>
+        View,
+
+        /// <summary>
+        /// Contains a value that indicates the processtype is translation.
+        /// </summary>
+        Translation
+    }
+
+    /// <summary>
+    /// This class represents a component preview transform within the system and is to be used for interaction with application user interfaces.
+    /// </summary>
+    public class ComponentPreviewTransformModel
+    {
+        /// <summary>
+        /// Gets or sets the identity of the component preview transform.
+        /// </summary>
+        public long ComponentPreviewTransformsId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the component preview transform.
+        /// </summary>
+        public string ComponentPreviewTransformName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the identity of the component type.
+        /// </summary>
+        public DateTime CreateDate { get; set; }
+
+        /// <summary>
+        /// Gets or sets the identity of the component type.
+        /// </summary>
+        public long ComponentTypeId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the review state information.
+        /// </summary>
+        public ComponentProcessType State { get; set; } = ComponentProcessType.Translation;
+
+        /// <summary>
```

Comment:
What's this?

Reply:
Must have been on crack at the time. removed